lib/cdo/properties.rb

```diff
@@ -41,25 +41,63 @@ def self.get_user_metrics()
 
 def fetch_metrics()
   Properties.get(:metrics)||{
-    'created_at'=>"2013-12-31T23:59:59+00:00",
-    'created_on'=>"2013-12-31",
-    'csedweek_organizers'=>0,
-    'csedweek_teachers'=>0,
-    'csedweek_entire_schools'=>0,
-    'csedweek_students'=>0,
-    'csedweek_countries'=>0,
-    'petition_signatures'=>0,
-    'lines_of_code'=>0,
+    'created_at'=>"2015-11-04T21:11:40+00:00",
```

Comment:
while you're in the neighborhood, can you add spaces around the || to conform to our style guide?